application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarkPaginatorInteractor.java

```diff
@@ -0,0 +1,480 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.JDA;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.concrete.ThreadChannel;
+import net.dv8tion.jda.api.entities.emoji.Emoji;
+import net.dv8tion.jda.api.events.interaction.ModalInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.command.GenericCommandInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.ButtonInteractionEvent;
+import net.dv8tion.jda.api.events.interaction.component.SelectMenuInteractionEvent;
+import net.dv8tion.jda.api.interactions.components.ActionRow;
+import net.dv8tion.jda.api.interactions.components.ComponentInteraction;
+import net.dv8tion.jda.api.interactions.components.LayoutComponent;
+import net.dv8tion.jda.api.interactions.components.buttons.Button;
+import net.dv8tion.jda.api.interactions.components.selections.SelectMenu;
+import net.dv8tion.jda.api.interactions.components.selections.SelectOption;
+
+import org.togetherjava.tjbot.commands.UserInteractionType;
+import org.togetherjava.tjbot.commands.UserInteractor;
+import org.togetherjava.tjbot.commands.componentids.ComponentIdGenerator;
+import org.togetherjava.tjbot.commands.componentids.ComponentIdInteractor;
+import org.togetherjava.tjbot.commands.utils.Arraylizable;
+import org.togetherjava.tjbot.db.generated.tables.records.BookmarksRecord;
+
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.StringJoiner;
+import java.util.function.Predicate;
+import java.util.function.UnaryOperator;
+
+public final class BookmarkPaginatorInteractor implements UserInteractor {
+
+    private static final int ENTRIES_PER_PAGE = 10;
+    private static final Emoji BUTTON_PREV_EMOJI = Emoji.fromUnicode("⬅");
+    private static final String BUTTON_PREV_NAME = "button-prevoius";
+    private static final Emoji BUTTON_NEXT_EMOJI = Emoji.fromUnicode("➡");
+    private static final String BUTTON_NEXT_NAME = "button-next";
+    private static final String BUTTON_REMOVE_LABEL = "Delete selected bookmarks";
+    private static final String BUTTON_REMOVE_NAME = "button-remove";
+    private static final String SELECTMENU_REMOVE_NAME = "selectmenu-remove";
+
+    private static final MessageEmbed NO_BOOKMARKS_EMBED = BookmarksSystem.createTinyEmbed("""
+            You don't have any bookmarks!
+            """, BookmarksSystem.COLOR_FAILURE);
```

Comment:
y r u using multi like string here? or whatever those are called, ur string is just a single like, there are many more places where u've done this, use normal strings

Reply:
Must be because of copy pasting